test/RewardsStreamerMP.t.sol

```diff
@@ -2954,21 +2995,22 @@ contract FuzzTests is StakeManagerTest {
     }
 
     function _expectUnstake(address account, uint256 amount) internal {
+        CheckVaultParams storage expectedAccountParams = expectedAccountState[account];
+        expectedAccountParams.account = vaults[account];
+        if (expectedVaultLockState[expectedAccountParams.account].lockEnd >= vm.getBlockTimestamp()) {
+            expectedRevert = IStakeManager.StakeManager__FundsLocked.selector;
+            console.log("locked");
+            return;
+        }
         if (amount == 0) {
             expectedRevert = StakeMath.StakeMath__InvalidAmount.selector;
             return;
         }
-        CheckVaultParams storage expectedAccountParams = expectedAccountState[account];
         if (amount > expectedAccountParams.stakedBalance) {
             expectedRevert = StakeMath.StakeMath__InsufficientBalance.selector;
             console.log("no balance");
```

Comment:
same here

Reply:
ah I see it's not a change. but I usually keep logs only while debugging, if it's something needed I would put it in the error message of an assertion

Reply:
Yup, can remove those while at it.